csun-metalab/nomi
# NAM-554 -- UX: Use same icon (preferably cog wheel) to open upload modal

# Description
Replaced `edit button` icons with `cog-wheel` .
  
# Testing
1. Login as `steve` .
2. Visit ***`student gallery`*** and ensure the icon is now a `cog-wheel` .
3. Ensure the hover animation is changed to a `spin` rather than `tilt` .
4. Repeat steps 2-3 with ***`student profile`*** .
